src/main/java/io/apicurio/datamodels/asyncapi/visitors/IAaiVisitor.java

```diff
@@ -16,93 +16,130 @@
+import io.apicurio.datamodels.asyncapi.models.AaiSchemaItems;
+import io.apicurio.datamodels.asyncapi.models.AaiSecurityRequirement;
+import io.apicurio.datamodels.asyncapi.models.AaiSecurityScheme;
+import io.apicurio.datamodels.asyncapi.models.AaiSecuritySchemeItems;
 import io.apicurio.datamodels.asyncapi.models.AaiServer;
 import io.apicurio.datamodels.asyncapi.models.AaiServerVariable;
+import io.apicurio.datamodels.asyncapi.models.AaiTag;
+import io.apicurio.datamodels.asyncapi.models.AaiTraitItem;
+import io.apicurio.datamodels.asyncapi.models.AaiTraitItems;
 import io.apicurio.datamodels.core.visitors.IVisitor;
 
 /**
  * A node visitor for the AsyncAPI data model.
  *
  * @author eric.wittmann@gmail.com
+ * @author Jakub Senko <jsenko@redhat.com>
  */
 public interface IAaiVisitor extends IVisitor {
 
-//    void visitAaiChannelItem(AaiChannelItem node);
-//
-//    void visitAaiChannelItems(AaiChannelItems node);
-//
-//    void visitAaiComponents(AaiComponents node);
-//
-//    void visitAaiContact(AaiContact node);
-//
-//    void visitAaiCorrelationIdItems(AaiCorrelationIdItems node);
-//
-//    void visitAaiCorrelationId(AaiCorrelationId node);
-//
-//    void visitAaiDocument(AaiDocument node);
-//
-//    void visitAaiExampleItems(AaiExampleItems node);
-//
-//    void visitAaiExternalDocumentation(AaiExternalDocumentation node);
-//
-//    void visitAaiHeaderItem(AaiHeaderItem node);
-//
-//    void visitAaiHeaderItems(AaiHeaderItems node);
-//
-//    void visitAaiInfo(AaiInfo node);
-//
-//    void visitAaiLicense(AaiLicense node);
-//
-//    void visitAaiMessageBase(AaiMessageBase node);
-//
-//    void visitAaiMessageItems(AaiMessageItems node);
-//
-//    void visitAaiMessage(AaiMessage node);
-//
-//    void visitAaiMessageTraitExtendedItem(AaiMessageTraitExtendedItem node);
-//
-//    void visitAaiMessageTraitItems(AaiMessageTraitItems node);
-//
-//    void visitAaiMessageTrait(AaiMessageTrait node);
-//
-//    void visitAaiOAuthFlow(AaiOAuthFlow node);
-//
-//    void visitAaiOAuthFlows(AaiOAuthFlows node);
-//
-//    void visitAaiOperationBase(AaiOperationBase node);
-//
-//    void visitAaiOperation(AaiOperation node);
-//
-//    void visitAaiOperationTraitExtendedItem(AaiOperationTraitExtendedItem node);
-//
-//    void visitAaiOperationTraitItems(AaiOperationTraitItems node);
-//
-//    void visitAaiOperationTrait(AaiOperationTrait node);
-//
-//    void visitAaiParameterItems(AaiParameterItems node);
-//
-//    void visitAaiParameter(AaiParameter node);
-//
-//    void visitAaiProtocolInfoItems(AaiProtocolInfoItems node);
-//
-//    void visitAaiProtocolInfo(AaiProtocolInfo node);
-//
-//    void visitAaiSchemaItems(AaiSchemaItems node);
-//
-//    void visitAaiSecurityRequirement(AaiSecurityRequirement node);
-//
-//    void visitAaiSecuritySchemeItems(AaiSecuritySchemeItems node);
-//
-//    void visitAaiSecurityScheme(AaiSecurityScheme node);
-//
+    void visitAaiChannelItem(AaiChannelItem node);
```

Comment:
I think these visit methods should not include the "Aai" in the name of the method.  So e.g. this one should be `visitChannelItem`.